config/opal_setup_wrappers.m4

```diff
@@ -220,18 +228,19 @@ EOF
 AC_DEFUN([OPAL_SETUP_RUNPATH],[
     OPAL_VAR_SCOPE_PUSH([LDFLAGS_save rpath_script rpath_outfile wl_fc])
 
-    AC_MSG_CHECKING([if linker supports RUNPATH])
     # Set the output in $runpath_args
     runpath_args=
     LDFLAGS_save=$LDFLAGS
     LDFLAGS="$LDFLAGS -Wl,--enable-new-dtags"
-    AC_LANG_PUSH([C])
-    AC_LINK_IFELSE([AC_LANG_PROGRAM([], [return 7;])],
-                   [WRAPPER_RPATH_SUPPORT=runpath
-                    runpath_args="-Wl,--enable-new-dtags"
-                    AC_MSG_RESULT([yes (-Wl,--enable-new-dtags)])],
-                   [AC_MSG_RESULT([no])])
-    AC_LANG_POP([C])
+    AS_IF([test x"$enable_wrapper_runpath" = x"yes"],
```

Comment:
Doesn't this change the default to only enable runpath if it is explicitly asked for? If you're changing the default, then you'll also need to update README. But I'm not sure that we want to change the default.

I thought you just wanted the ability to disable runpath for some specific platforms at LANL...?